src/controllers/import.js

```diff
@@ -126,6 +132,38 @@ function ImportController(context) {
     };
   }
 
+  /**
+   * Get all import jobs between startDate and endDate
+   * @param {object} requestContext - Context of the request.
+   * @param {string} requestContext.params.startDate - The start date of the range.
+   * @param {string} requestContext.params.endDate - The end date of the range.
+   * @param {string} requestContext.pathInfo.headers.x-import-api-key - API key to use for the job.
```

Comment:
This will need to be updated once we merge in scoped API key support.

```suggestion
   * @param {string} requestContext.pathInfo.headers.x-api-key - API key to use for the job.
```